### PR TITLE
Add Blockchain video lightbox and downloads

### PR DESCRIPTION
**Pull request by kelbyhawn**
Added the `display_video_thumbnail` video player and downloads to video series on https://code.org/blockchain.

**Jira ticket:** [ACQ-779](https://codedotorg.atlassian.net/browse/ACQ-779)

----

https://github.com/code-dot-org/code-dot-org/assets/9256643/6ec45e8d-da83-4fa4-9018-051b6d4044c6

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/blockchain_vid_01_intro.haml
Comment (davidsbailey) [inline pegasus/sites.v3/code.org/views/blockchain_vid_01_intro.haml:1]: I count 214 haml files in this directory. if we could please put it onto our list to organize some of these new files into subdirectories that would be great :-)

**Review by davidsbailey — APPROVED** [review]

File: pegasus/sites.v3/code.org/views/blockchain_vid_01_intro.haml
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/views/blockchain_vid_01_intro.haml:1]: Yeah! I'll add a `blockchain` subdirectory now and will create a backlog ticket to further organize things.

Comment (davidsbailey) [reply pegasus/sites.v3/code.org/views/blockchain_vid_01_intro.haml:1]: Perfect!

Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/views/blockchain_vid_01_intro.haml:1]: Here's the ticket for posterity: https://codedotorg.atlassian.net/browse/ACQ-791

**Pull request merged**